### PR TITLE
Add prisma as format

### PR DESCRIPTION
**Pull request by pat-trunk-io**
Enable prisma format

Linter depends on a tool, and requires latest beta to work.

### CODE REVIEW

**Review by laurit17 — APPROVED** [review]
First linter with tool dep!!! 🥳

File: linters/prisma/plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 0.1
+lint:
+  definitions:
+    - name: prisma
+      tools: [prisma]
+      files: [prisma]
+      known_good_version: 4.16.1
```
Comment (TylerJang27) [inline linters/prisma/plugin.yaml:7]: We'll want to add in `suggest_if: never` (maybe eventually "if_files") for explicitness

**Review by TylerJang27 — COMMENTED** [review]
See suggest_if

File: linters/prisma/plugin.yaml (hunk 1 above)
Comment (pat-trunk-io) [reply linters/prisma/plugin.yaml:7]: Done

**Review by TylerJang27 — APPROVED** [review]

**Comment by pat-trunk-io** [comment]
Current dependencies on/for this PR:
* main
  * **PR #343** <a href="https://app.graphite.dev/github/pr/trunk-io/plugins/343" target="_blank">Graphite</a>  👈
    * **PR #344** <a href="https://app.graphite.dev/github/pr/trunk-io/plugins/344" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/trunk-io/plugins/343?utm_source=stack-comment).

**Pull request merged**